Add elastic-agent min requirements (#1579)
elastic/observability-docs: docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc

* {eck-ref}/k8s-elastic-agent.html[Run {agent} on ECK] -- for {eck} users
--

> == Minimum Requirements

Reviewer: ```suggestion
[discrete]
== Minimum Requirements
```

> An example of the {agent} resource consumption is below.

Reviewer: The statement here seems too noncommittal for docs. The content doesn't live up to the expectation set by the heading.

We should start with a clear statement of minimum system requirements. If it's not possible to make a clear statement, then I think we need to do more testing before we try to document these requirements. IMO if the requirements vary by platform, then we need to provide a breakdown by platform in table format.

Also we need to provide more guidance about how much extra memory is consumed when you add integrations. Otherwise I feel this content is not super useful and potentially misleading.

Can you recast this section so that it 1) Makes a direct statement about minimum system requirements, and 2) provides an example of system resources used by a specific instance?

The {agent} used the default policy, running the system integration and self-monitoring.

> + **CPU** Under 2% total, including all monitoring processes.

Reviewer: ```suggestion
+ **CPU** Under 2% total, including all monitoring processes
```

+ **Disk** 640MB
+ **RSS Mem Size** 300 MB

> Adding additional integrations will increase the memory used by the agent and it's processes.

Reviewer: ```suggestion
Adding integrations will increase the memory used by the agent and its processes.
```

* {eck-ref}/k8s-elastic-agent.html[Run {agent} on ECK] -- for {eck} users
--

[discrete]
== Minimum Requirements

Minimum requirements have been determined by running the {agent} (`v8.0.0`) on a GCP `e2-micro` instance (2vCPU/1GB).
The {agent} used the default policy, running the system integration and self-monitoring.

> + **CPU** Under 2% total, including all monitoring processes
> + **Disk** 640MB
> + **RSS Mem Size** 300 MB
> 

Reviewer: ```suggestion
|===
| **CPU** | Under 2% total, including all monitoring processes
| **Disk** | 640MB
| **RSS Mem Size** | 300 MB
|===
```

Reviewer: The formatting here looks wrong in the output. Sorry I didn't notice this before! I've added markup for a table, but you could format this as a list instead.
